bmaptools/TransRead.py

```diff
@@ -589,6 +590,12 @@ def _print_warning(timeout):
         username = parsed_url.username
         password = parsed_url.password
 
+        if not username and not password:
+            auth = netrc.netrc().authenticators(parsed_url.hostname)
+            if auth:
+                username = auth[0]
+                password = auth[2]
```

Comment:
I am confused, I thought I merged this patch

Reply:
You closed it without merging it..... I didn't really intent to open a new PR; I didn't realize that would happen when I clicked the button.

Anyway it doesn't pass CI, so it needs a new revision anyway (yay CI!)

Reply:
@JPEWdev  oh, sorry, I did not realize this. Let me try again.